sql/create.sql

```diff
@@ -614,6 +614,15 @@ begin
     cache text not null
   );
 
+  create table if not exists app_owner_details
+  (
+    id varchar(128) not null,
+    owner_name text not null,
+    owner_email text not null,
+    app uuid references apps("app"),
+    owner_organization text not null
```

Comment:
Not too sure about this column, since we already have the organization as part of the app do we need a separate organization for the owner?

Reply:
Updated and taking references from Organization.

Reply:
Updated and taking references from Organization.